MiniShopping/MiniShopping/Supporting Files/AppDelegate.m

```diff
@@ -0,0 +1,33 @@
+#import "AppDelegate.h"
+
+@interface AppDelegate ()
+
+@end
```

Comment:
Empty extension over `AppDelegate` should be removed. Before sending changes for final review, please self-review the code and remove the codes which is not needed.